src-vue/src/views/PlayView.vue

```diff
@@ -32,6 +39,11 @@ export default defineComponent({
             <div v-if="northstarVersion !== ''" class="fc_changelog__link" @click="showChangelogPage">
                 (see patch notes)
             </div>
+            <br />
```

Comment:
You should apply spacing with CSS rather than using this ugly tag :)

Reply:
My CSS is a bit rusty, do you maybe wanna do that part? ^^"

Reply:
Yay! Gotta find a good way to display stats first though.

Reply:
@GeckoEidechse still need help for adding margin to this? :)

Reply:
Uh, yes please ^^"